install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -70,6 +58,10 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 						Search:      "open-vsx.org",
 						Replacement: openVSXProxyUrl,
 						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
+					}, {
+						Search:      "https://raw.githubusercontent.com/gitpod-io/openvsx.org.json/main/marketplace.json",
```

Comment:
I think we should use `https://ide.gitpod.io/code/markeplace` now in `product.json` instead of the github url so it's not misleading

Reply:
@iQQBot could you advise here since URL will be different for Self-Hosted?

Reply:
Thanks for the suggestion. I added it in the latest PR commit.